### ISSUE TITLE
navigation buttons in collected material

### ISSUE THREAD

**Issue by urapadmin** [urapadmin/kiosk#402]
### @test 9.6.2
Does not strike me the most elegant implementation but there they are: left and right arrows on the collected material form to navigate through the collected materials of the current locus.

**Comment by lbestock** [comment]
I went here after testing #401. What I did was click on CM for the locus I was in at the time. It had no CM so was blank. I then hit the navigation buttons next to context in the banner and it moved me through contexts until I found one that did have CM so I could test it. I hope that is how you intended the buttons of #401 to work - it's great! Once into CM the buttons of this ticket work fine, though I agree that their location makes them a bit awkward - they are hard to hit properly on the iPad mini. A suggestion: can we put more space between the + and the left and right arrows? I twice created new CM when I was trying to go through the list of what was already there.

**Comment by urapadmin** [comment]
I am not sure I entirely understand the first description. #401 lets you step through the loci of a unit, #402 lets you step through the collected materials of a locus. I guess, that is what you meant. 

- [ ] I spaced out the buttons a bit more in cm and locus 
- [ ] and there is a question now when clicking on the + buttons on these forms

**Comment by lbestock** [comment]
I don't have it yet, but I understand what you mean. In stepping through the loci of  a unit, if one is in the CM tab, one sees which loci have CM at all and which do not, which is useful. One stays in the tab one is in while moving through loci, so one can see which have what. 

I can't look at the spacing or question yet but it does occur to me that one of my confusions might have been ordering of the buttons. In CM the navigation buttons are ordered + left right. In loci they are ordered Left + Right. They should be the same; I came to CM right after loci and was thus used to that. To me the latter makes more sense - at least, I did not accidentally create loci in the same way I did CM.
